src/python/impactx/dashboard/Analyze/plot_PhaseSpaceProjections/phaseSpace.py

```diff
@@ -57,12 +55,12 @@ def run_simulation():
     ref = pc.ref_particle()
     ref.set_charge_qe(-1.0).set_mass_MeV(0.510998950).set_kin_energy_MeV(kin_energy_MeV)
 
-    distr = distribution_parameters()
```

Comment:
@ax3l , Would renaming the variables like this work? Maybe this goes against naming conventions and there is a better name we can give for the variables?

Reply:
@proy30 can you tell me again renamed from which to which name? Because of the double change, I cannot see anymore what `distr = distribution_parameters()` was renamed into.

Reply:
'distr' changed to 'distribution' 
'fodo' changed to 'lattice_configuration'

Reply:
ah, that works well here. The other names were a specific example setup.